### ISSUE TITLE
HTML <main> Tag missing

### ISSUE THREAD

**Issue by piwysocki** [auth0/lock#2113]
### Describe the problem

Page must have means to bypass repeated blocks. 
Page is not compliant with WCAG 2.0 level A

[rule details](https://dequeuniversity.com/rules/axe/4.4/bypass?application=AxeChrome)

### What was the expected behavior?

<main> tag should be added to document.

### Reproduction

rendered login page:

![obraz](https://user-images.githubusercontent.com/86244209/163573875-ed71bbd7-ffd4-4955-9dba-ab25a6da6467.png)

highlighted <div> should be replaced by <main>

### Environment

- Version of this library used: 11.32.2
